src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: string | 'red' | 'green' | 'blue';
+  getArea() : number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: 'triangle',
+  ) {
+    this.shape = 'triangle';
 
-}
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('your error message');
```

Comment:
Check 3 point in the [checklist](https://github.com/mate-academy/ts_figures_classes/blob/master/checklist.md)

Change error message in all places